docs/release_process.md

```diff
@@ -115,50 +120,52 @@ E.g., the following `2022.08.02` tag includes only `change-1` and `change-2`.
 ```
 
 ### Create Release Notes on GitHub
+
 After you push the tag to GitHub, you should also
 [make a pre-release on GitHub][github-new-release] for the tag.
 
 1. Choose the tag you just pushed (e.g., `2022.08.02`)
 2. Type the same tag name for the releae title (e.g., `2022.08.02`)
 3. Click "Previous tag:" and choose the tag currently on production.
-   * You can find this at [the `__version__` endpoint][prod-version].
+   - You can find this at [the `__version__` endpoint][prod-version].
 4. Click the "Generate release notes" button!
 5. Check the pre-release box.
 6. Click "Publish release"
 
-
 ## Release to Prod
+
 We leave the tag on [Stage][stage] for a week so that we (and especially QA)
 can check the tag on GCP infrastucture before we deploy it to production. To
 deploy the tag to production:
 
 1. File an [SRE ticket][sre-board] to deploy the tag to [Prod][prod].
-   * Include a link to the GitHub Release
-   * You can assign it directly to our primary SRE for the day
+   - Include a link to the GitHub Release
+   - You can assign it directly to our primary SRE for the day
 2. When SRE starts the deploy, "cloudops-jenkins" will send status messages
    into the #fx-private-relay-eng channel.
 3. When you see `PROMOTE PROD COMPLETE`, do some checks on prod:
-   * Check sentry prod project for a spike in any new issues
-   * Check grafana dashboard for any unexpected spike in ops
-   * Spot-check the site for basic functionality
-   * Ping SDET to run end-to-end tests on prod
+   - Spot-check the site for basic functionality
+   - Check [sentry prod project](https://mozilla.sentry.io/releases/?environment=prod) for a spike in any new issues
+   - Check [grafana dashboard](https://earthangel-b40313e5.influxcloud.net/d/qiwPC76Zk/fx-private-relay?orgId=1&refresh=1m&from=now-1h&to=now) for any unexpected spike in ops
+   - [Run end-to-end tests](https://github.com/mozilla/fx-private-relay/actions/workflows/playwright.yml) on prod
```

Comment:
These have not run successfully for months. Maybe we should note that they are broken?